tests/python/gpu/test_kvstore_gpu.py

```diff
@@ -21,7 +21,7 @@
 import mxnet as mx
 import numpy as np
 import unittest
-from mxnet.test_utils import assert_almost_equal, default_context
+from mxnet.test_utils import assert_almost_equal, default_context, list_gpus
```

Comment:
list_gpus in utils is a bad way to identify the GPU's, better use [context.num_gpus](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/context.py#L244)